tests/frame/join_test.py

```diff
@@ -108,13 +107,14 @@ def test_full_join_duplicate(constructor: Constructor) -> None:
     df_left = nw_main.from_native(constructor(df1)).lazy()
     df_right = nw_main.from_native(constructor(df2)).lazy()
 
-    # polars throws `NarwhalsError`, everything else should raise `DuplicateError`
-    exceptions = [DuplicateError, nw.exceptions.NarwhalsError]
-
+    exceptions: list[type[Exception]] = [nw.exceptions.NarwhalsError]
```

Comment:
@FBruzzesi @tylerriccio33 just fyi, we can catch polars' DuplicateError and raise it as NarwhalsError, so now it's (mostly) unified